src/test/java/com/wikia/webdriver/testcases/auth/LoginTests.java

```diff
@@ -230,12 +231,17 @@ private SignInPage navigateToSignInOnMobile(TopBar topBar) {
       .navigateToSignIn();
   }
 
-  private SignInPage openLoginModalOnDesktop() {
-    return new DetachedSignInPage(new NavigationBar().clickOnSignIn());
+//  private SignInPage openLoginModalOnDesktop() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)